tests/test_layers_convolution.py

```diff
@@ -61,9 +61,17 @@
 n = tl.layers.DeConv2d(nin, n_filter=32, filter_size=(3, 3), strides=(2, 2), name='DeConv2d')
 print(n)
 shape = n.outputs.get_shape().as_list()
+print(shape[1:])
```

Comment:
Modifying tests now will collide with my PR: https://github.com/tensorlayer/tensorlayer/pull/494

Keeping them and updating them later on would be a good idea.